.env.example

```
@@ -84,13 +84,21 @@ AUTHENTIK_API_TOKEN=
 # Required for /create-user-accounts and /invite-outline-user.
 # For Outline Cloud, keep the default https://app.getoutline.com.
 # For self-hosted Outline, use https://your-outline-host.
 OUTLINE_BASE_URL=https://app.getoutline.com
 OUTLINE_API_TIMEOUT_SECONDS=20.0
-OUTLINE_API_KEY=
+OUTLINE_ADMIN_API_KEY=
+# Optional for /wiki. Create this key for a dedicated regular Outline Member
+# with access only to member-safe collections; scope it to documents.search and
+# stars.list. OUTLINE_WIKI_API_KEY is accepted as a legacy fallback. That
+# account's starred documents are /wiki quick links.
+OUTLINE_DISCORD_MEMBER_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OUTLINE_DISCORD_MEMBER_API_KEY key should go before the WEBHOOK_SHARED_SECRET key

(UnorderedKey)